src/style.css

```diff
@@ -0,0 +1,44 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  background-color: #fff;
+  width: fit-content;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
```

Comment:
the outermost star should not be indented
![image](https://github.com/mate-academy/layout_stars/assets/104684446/77519f67-d4e6-41ff-b5ef-9552898f7ef2)